frontend/src/components/AgreementCard.tsx

```diff
@@ -42,76 +44,110 @@ const AgreementCard: FC<IAgreementCardProps> = ({
-                sx={{ background: generateColorFromString(category || '', true), color: 'white' }}
-                label={category}
-              />
-            )}
-          </Box>
-          <Stack flexDirection="row" justifyContent="center" paddingY={2}>
-            <DescriptionOutlinedIcon sx={{ fontSize: 52 }} color="disabled" />
-          </Stack>
-        </CardContent>
-        <CardContent>
-          <Stack>
-            <Stack
-              flexDirection="row-reverse"
-              justifyContent="space-between"
-              alignItems="center"
-              paddingBottom={1}
-            >
-              <DropDownMenu
-                mainIcon={<MoreHorizIcon />}
-                name="agreement-menu"
-                menuItems={[
-                  {
-                    text: t(isArchived ? StringBank.UNARCHIVE : StringBank.ARCHIVE),
-                    icon: <Inventory2OutlinedIcon />,
-                    action: () => archiveAgreement(id, !isArchived),
-                  },
-                ]}
-              />
-              <Typography
-                fontWeight="bold"
-                variant="body1"
-                overflow="hidden"
-                textOverflow="ellipsis"
-                whiteSpace="nowrap"
-                maxWidth="calc(100% - 40px)"
+    <>
+      <MainCard onClick={() => navigate(`/${slug}/agreement/${id}`)}>
+        <Stack>
+          <CardContent sx={{ backgroundColor: cardBackgroundColor, position: 'relative' }}>
+            <Box position="absolute" left="1rem" right="1rem">
+              {category && (
+                <Chip
+                  sx={{ background: generateColorFromString(category || '', true), color: 'white' }}
+                  label={category}
+                />
+              )}
+            </Box>
+            <Stack flexDirection="row" justifyContent="center" paddingY={2}>
+              <DescriptionOutlinedIcon sx={{ fontSize: 52 }} color="disabled" />
+            </Stack>
+          </CardContent>
+          <CardContent>
+            <Stack>
+              <Stack
+                flexDirection="row-reverse"
+                justifyContent="space-between"
+                alignItems="center"
+                paddingBottom={1}
               >
-                {title}
+                <DropDownMenu
+                  mainIcon={<MoreHorizIcon />}
+                  name="agreement-menu"
+                  menuItems={[
+                    {
+                      text: t(isArchived ? StringBank.UNARCHIVE : StringBank.ARCHIVE),
+                      icon: <Inventory2OutlinedIcon />,
+                      action: () => archiveAgreement(id, !isArchived),
+                    },
+                    {
+                      text: 'Delete',
+                      icon: <DeleteOutlineIcon sx={{ color: 'red' }} />,
+                      action: () => {
+                        handleClickOpenDialog();
+                      },
+                    },
```

Comment:
@yishain11 
- match figma trash bin icon and trash bin color 
- match figma text `color`

Reply:
1. I need an icon from Tsur for that.
2. Done

Reply:
@yishain11 
1. https://zpl.io/p1K4jwr

Reply:
done